connector/saml/saml.go

```diff
@@ -72,6 +75,18 @@ func init() {
 	}
 }
 
+// normalizeBinding verifies binding from config and normalize the string
+func normalizeBinding(binding string) (string, error) {
+	binding = strings.ToLower(strings.TrimSpace(binding))
+	if binding == "" {
+		return connector.SAMLBindingPOST, nil
+	}
+	if binding != connector.SAMLBindingPOST && binding != connector.SAMLBindingRedirect {
```

Comment:
probably just use a globally defined map + a lookup

Reply:
only two string const comparisons. I'm trying to avoid a global var. Anyway, I will add a global map.